src/appliance-onboarding-script/appliance_setup/avs/avsarconboarder/utils/user_config_validatator.py

```diff
@@ -2,7 +2,6 @@
 from avs.avsarconboarder.exception import InvalidInputError
 from avs.avsarconboarder.retriever.cloud_data.cloud_data_retriever import CloudDataRetriever
 
-
```

Comment:
If this change is not required, can we remove it from the review?

Reply:
sure